lib/LaTeXML/resources/CSS/LaTeXML.css

```diff
@@ -191,7 +191,7 @@ a.ltx_LaTeXML_logo { text-decoration: none; }
 
 .ltx_tabular .ltx_tabular { width:100%; }
 .ltx_inline-block,
-.ltx_inline-logical-block { display:inline-block; }
+.ltx_inline-logical-block { display:inline-block; text-indent:0px; }
```

Comment:
Right, I have seen text-indent getting inherited and cause issues! This will have some side effects, but all good I hope. It's just a bit weird to see it in a xypic PR.

Reply:
Indeed, but not as weird as the effect it caused on a test case! :>